tests/Elasticsearch/Tests/YamlRunnerTest.php

```diff
@@ -171,7 +171,7 @@ public function testAsyncIntegration($testProcedure, $skip, $setupProcedure, $fi
         }
 
         if (null !== $setupProcedure) {
-            $this->processProcedure(current($setupProcedure), 'setup');
+            $this->processProcedure(current($setupProcedure), 'setup', $fileName);
```

Comment:
This one is related to recent change https://github.com/elastic/elasticsearch-php/commit/51b9b9b0981d8021923833f0bb39011fb30bb631 - and it would be discover immediately